lib/internal/Magento/Framework/File/Uploader.php

```diff
@@ -25,6 +25,9 @@
  */
 class Uploader
 {
+    // Directory and filename must be no more than 255 characters in length
+    protected const MAX_PATH_FILENAME_LENGTH = 255;
```

Comment:
Unfortunately adding new constant isn’t treated as backward compatible change, could you make it private in order to fix this issue? If needed - you could add the same constant to children classes

Reply:
I'll do.